Source/KeyboardManager.swift

```diff
@@ -5,109 +5,160 @@
 
 import UIKit
 
-public typealias KeyboardManagerEventClosure = (KeyboardManagerEvent, KeyboardManagerEventData) -> Void
+public typealias KeyboardManagerEventClosure = (KeyboardManagerEvent) -> Void
 
 public enum KeyboardManagerEvent {
-    case willShow
-    case didShow
-    case willHide
-    case didHide
-}
+    case willShow(KeyboardManagerEvent.Data)
+    case didShow(KeyboardManagerEvent.Data)
+    case willHide(KeyboardManagerEvent.Data)
+    case didHide(KeyboardManagerEvent.Data)
 
-public struct KeyboardManagerEventData {
     public struct Frame {
         var begin: CGRect
         var end: CGRect
     }
-    var frame: Frame
-    var animationCurve: String
-    var animationDuration: Double
-    var isLocal: Bool
-
-    static func null() -> KeyboardManagerEventData {
-        let frame = Frame(begin: CGRect.zero, end: CGRect.zero)
-        return KeyboardManagerEventData(frame: frame, animationCurve: "", animationDuration: 0.0, isLocal: false)
+
+    public struct Data {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/alphatroya/KeyboardManager/blob/b88ef862088cfe316885e971372c9fbbf52f8dcc/Source/KeyboardManager.swift#L21"></span>Public declarations should be documented.</td>
    </tr>
  </tbody>
</table>